APOLLO-28028: adds connector-plugin-service description (#142)
lucidworks/fusion-cloud-native: survival_guide/1_concepts.adoc

|`connectors-rpc` |gRPC |Deployment |`analytics` or `system` |Yes (CPU or custom metric) |gRPC service for managing SDK-based connector plugins.

> |`connector-plugin-service-<connector_plugin>` | HTTP/TCP |Deployment |`analytics` or `system` |Yes (CPU or custom metric) | Running instance of each connector plugin. *Note:* There is a base deployment `connector-plugin-service` with 0 replicas which is used for deploying each connector plugin. It should not be deleted or scaled.

Reviewer: ```suggestion
|`connector-plugin-service-<connector_plugin>` | HTTP/TCP |Deployment |`analytics` or `system` |Yes (CPU or custom metric) | Deployment for each connector plugin type. *Note:* There is a base deployment, `connector-plugin-service` with 0 replicas. This is used as a deployment template for each connector plugin type. It should not be deleted or scaled.
```
